orchestrator/careplancontributor/service.go

```diff
@@ -67,9 +72,18 @@ func (s Service) withSession(next func(response http.ResponseWriter, request *ht
 	}
 }
 
+func (s Service) handleProxyToEPD(writer http.ResponseWriter, request *http.Request, session *user.SessionData) {
+	clientFactory := clients.Factories[session.FHIRLauncher](session.Values)
+	proxy := coolfhir.NewProxy(log.Logger, clientFactory.BaseURL, "/contrib/ehr/fhir", clientFactory.Client)
+	proxy.ServeHTTP(writer, request)
+}
+
 // handleGetPatient is the REST API handler that returns the FHIR Patient.
 func (s Service) handleGetPatient(response http.ResponseWriter, request *http.Request, session *user.SessionData) {
-	fhirClient := coolfhir.ClientFactories[session.FHIRLauncher](session.Values)
+	// TODO: Remove this when frontend works on the proxy endpoint
```

Comment:
This specific information is currently extracted from the orca session and unknown to the frontend before fetching the Patient that has been launched. Do you reckon we pass this onto the frontend on redirect?